packages/next/server/config-shared.ts

```diff
@@ -81,7 +81,7 @@ export type NextConfig = { [key: string]: any } & {
   pageExtensions?: string[]
   compress?: boolean
   poweredByHeader?: boolean
-  images?: ImageConfig
+  images?: Partial<ImageConfig>
```

Comment:
It looks like we need to update the `NextConfigComplete` type to not treat this as a partial since these values should be defined after being normalized.